### PR TITLE
pkgs/tools: pkgconfig -> pkg-config (2)

### PR DESCRIPTION
**Pull request by siraben**
###### Motivation for this change
Breaking up the change into ~1000 files changed PRs makes it more manageable.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/package-management/apk-tools/default.nix [hunk 1]
```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
     "MANDIR=$(out)/share/man"
     "DOCDIR=$(out)/share/doc/apk"
     "INCLUDEDIR=$(out)/include"
-    "PKGCONFIGDIR=$(out)/lib/pkgconfig"
+    "PKGCONFIGDIR=$(out)/lib/pkg-config"
```
Comment (SuperSandro2000) [inline pkgs/tools/package-management/apk-tools/default.nix:22]: I think this is an accident.

File: pkgs/tools/system/minijail/default.nix [hunk 2]
```diff
@@ -40,11 +40,11 @@ stdenv.mkDerivation rec {
   installPhase = ''
     ./platform2_preinstall.sh ${version} $out/include/chromeos
 
-    mkdir -p $out/lib/pkgconfig $out/include/chromeos $out/bin \
```
Comment (SuperSandro2000) [inline pkgs/tools/system/minijail/default.nix:43]: Those are probably also not indented.

File: pkgs/tools/typesetting/ted/default.nix [hunk 3]
```diff
@@ -10,11 +10,11 @@ stdenv.mkDerivation rec {
   };
 
   preConfigure = ''
-    mkdir pkgconfig-append
-    pushd pkgconfig-append
+    mkdir pkg-config-append
```
Comment (SuperSandro2000) [inline pkgs/tools/typesetting/ted/default.nix:13]: Same here.

File: pkgs/tools/package-management/apk-tools/default.nix (hunk 1 above)
Comment (siraben) [reply pkgs/tools/package-management/apk-tools/default.nix:22]: Thanks.

File: pkgs/tools/system/efivar/default.nix [hunk 4]
```diff
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
     "bindir=$(bin)/bin"
     "mandir=$(man)/share/man"
     "includedir=$(dev)/include"
-    "PCDIR=$(dev)/lib/pkgconfig"
```
Comment (SuperSandro2000) [inline pkgs/tools/system/efivar/default.nix:56]: I think this is a mistake.

File: pkgs/tools/system/minijail/default.nix [hunk 5]
```diff
@@ -40,11 +40,11 @@ stdenv.mkDerivation rec {
...
+    mkdir -p $out/lib/pkg-config $out/include/chromeos $out/bin \
         $out/share/minijail
 
     cp -v *.so $out/lib
-    cp -v *.pc $out/lib/pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/tools/system/minijail/default.nix:47]: Another one.

**Comment by siraben** [comment]
@SuperSandro2000 when you merge these treewide PRs please also merge master→staging-next→staging to reduce the chance of merge conflicts there.

**Comment by Mic92** [comment]
merged to staging-next/staging as well.

**Pull request merged**